atom/browser/api/atom_api_app.h

```diff
@@ -50,6 +50,7 @@ class App : public AtomBrowserClient::Delegate,
   void OnCreateWindow(const GURL& target_url,
                       const std::string& frame_name,
                       WindowOpenDisposition disposition,
+					  const std::vector<base::string16>& features,
```

Comment:
Indenting again, maybe your editor is configured from tabs / spaces incorrectly?